crates/store/re_chunk/src/chunk.rs

```diff
@@ -705,6 +705,22 @@ impl Chunk {
         }
     }
 
+    /// Unconditionally inserts an [`ArrowListArray`] as a component column.
```

Comment:
Does unconditionally imply that it will replace the component if it already exists?

Reply:
Yes. I'll expand the comment.